docs/docsite/rst/reference_appendices/python_3_support.rst

```diff
@@ -1,3 +1,5 @@
+.. _ansible_and_python_3:
```

Comment:
I think this label should point to `docs/docsite/rst/dev_guide/developing_python_3.rst` (which already has the label `developing_python_3`) and not to this document.